TS.fsx

```diff
@@ -831,6 +831,10 @@ module Emit =
         Pt.Printl "}"
         Pt.Printl ""
 
+    let EmitElementTagNameMap () =
+        Pt.Printl "interface ElementTagNameMap extends HTMLElementTagNameMap, SVGElementTagNameMap { }"
```

Comment:
Should extend only `HTMLElementTagNameMap`. If not, we can't define other SVG elements such as SVGAElement by extending the interface. So `ElementTagNameMap` should be just an alias of `HTMLElementTagNameMap`.

```
extends HTMLElementTagNameMap { }
```

Reply:
This is a reasonable compromise as you said.

Reply:
And `ElementTagNameMap` should be deprecated.